Surface_mesh_approximation/include/CGAL/Variational_shape_approximation.h

```diff
@@ -1778,7 +1778,7 @@ class Variational_shape_approximation {
   }
 
   /*!
-   * @brief subdivides a chord recursively in range `[@a chord_begin, @a chord_end).`
+   * @brief subdivides a chord recursively in range `[chord_begin, chord_end).`
    * @param chord_begin begin iterator of the chord
    * @param chord_end end iterator of the chord
```

Comment:
This problem is already addressed in #5102 and in #5120 there is an essential part of the fix missing i.e. bringing the `.` outside of the backtics.

Reply:
I knew I saw that fix already :)

Reply:
OK I'll close my proposed pull request #5102 in favor of this pull request